addons/web_editor/static/src/js/backend.js

```diff
@@ -206,7 +206,10 @@ var FieldTextHtml = AbstractField.extend({
         };
         window.odoo[this.callback+"_updown"] = null;
         window.odoo[this.callback+"_downup"] = function (value) {
-            self.set_value(value);
+            //update the field value only if lang = en_US
+            if (self.lang === 'en_US') {
+                self._setValue(value);
+            }
```

Comment:
Before saas16, we were using internal_set_value method, which was just updating a value without re rendering, while in new view this method is no more exist and calling _setValue method on each key press, was updating the original value (follow by the write method)
regardless of the language, hence source translation terms was getting broken (when user change a content in other lang)